DQM/DTMonitorModule/src/DTChamberEfficiency.cc

```diff
@@ -79,7 +79,7 @@ DTChamberEfficiency::DTChamberEfficiency(const ParameterSet& pSet)
   edm::ConsumesCollector iC = consumesCollector();
 
   theMeasurementExtractor = new MuonDetLayerMeasurements(thedt4DSegments,thecscSegments,
-  							 labelRPCRecHits,labelGEMRecHits,labelME0RecHits,iC,true,false,false,false,false); 
+  							 labelRPCRecHits,InputTag(),InputTag(),InputTag(),iC,true,false,false,false,false); 
```

Comment:
The unused member variables should be removed from the class definition in `DTChamberEfficiency.h`